src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -120,8 +120,11 @@ case class WorkflowActor(workflow: WorkflowDescriptor,
   private def startRunnableCalls(): State = {
     tryStartingRunnableCalls() match {
       case Success(entries) =>
-        if (entries.nonEmpty) tryStartingRunnableCalls()
-        goto(WorkflowRunning)
+        if (entries.nonEmpty) {
+          startRunnableCalls()
```

Comment:
This shouldn't produce duplicate starting 'starting call' messages, though those messages leave out shard information so some might appear as duplicates